develop.html.mako

```diff
@@ -27,24 +27,73 @@ on getting started.
 <a name="developers"></a>
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where can you find the Developers of SqlAlchemy ?</p>
+
+<p><em>The Developers of SqlAlchemy kindly request all people participating in 
+these channels respect the <a href="/codeofconduct.html">Code of Conduct</a> when 
+seeking or providing support.</em></p>
 
 <ul>
-    <li>The <a href="/support.html#mailinglist">main
-    mailing list</a> is the most common - core devs assist users with issues of all kinds.</li>
-    <li>Developer discussion takes place
-    on the <a href="http://groups.google.com/group/sqlalchemy-devel">development mailing list</a>,
-    also hosted on Google groups.  Anyone can join this
-    list, though we ask that discussion be limited to architectural and developmental issues only.
-    </li>
-    <li>Some development discussion occurs on the
-    <code>#sqlalchemy-devel</code> IRC channel on Freenode.
-    This channel is <strong>only</strong> for users that are interested in contributing code,
-    tests, documentation, or other developmental resources, and no support issues will be
-    addressed here.   IRC-based support is available on the
-    <a href="/support.html#irc">main IRC channel</a>.</li>
+    <li>Mailing Lists</li>
+    <ul>
+        <li>The <a href="/support.html#mailinglist">main mailing list</a> is the most 
+            common - core devs assist users with issues of all kinds. All SqlAlchemy 
+            Users and Third-Party Library authors are encouraged to participate here
+            and seek support of any kind.
+            </li>
+
+        <li>A secondary <a href="https://groups.google.com/group/sqlalchemy-devel">
+            development mailing list</a>, also hosted on Google groups, is dedicated
+            to Developer concerns such as: release announcements, architectural and 
+            developmental issues, and issues pertaining to Third-Party Library authors.
+            This list is typically low traffic; all posts intended for this channel are
+            welcome on the <a href="/support.html#mailinglist">main mailing list</a> .
+            </li>
+    </ul>
+    <li>Real-Time Communication</li>
+    <ul>
+        <li>Development discussion continually occurs in the 
+            <a href="https://gitter.im/sqlalchemy/devel">sqlalchemy/devel</a> Gitter room
+            througout the week. Discussion in this room is intended for users 
+            who are interested in contributing code, tests, documentation, or other 
+            developmental resources. This channel is the current preferred real-time
+            discussion medium for SqlAlchemy contributors and hosts the 
+            <a href="#scheduled-developer-meetings">Scheduled Developer Meetings</a>.
+            Gitter based support is available via the
+            <a href="https://gitter.im/sqlalchemy/devel">sqlalchemy/community</a> Gitter
```

Comment:
Type here: https://gitter.im/sqlalchemy/devel -> https://gitter.im/sqlalchemy/community

Reply:
great catch! thank you!